statsmodels/compat/pandas.py

```diff
@@ -2,47 +2,21 @@
 
 from distutils.version import LooseVersion
 
-import numpy as np
 import pandas
+from pandas import RangeIndex, Float64Index  # noqa:F401
```

Comment:
Ideally, anything that not needed from compat should be directly imported from the package that provides it.